src/main/scala/org/broadinstitute/dsde/agora/server/webservice/util/GoogleApiUtils.scala

```diff
@@ -17,7 +17,7 @@ object GoogleApiUtils {
   val emailAddress = AgoraConfig.gcsServiceAccountEmail
   val JSON_FACTORY = JacksonFactory.getDefaultInstance
   val httpTransport = GoogleNetHttpTransport.newTrustedTransport
-  val directoryScopes = Collections.singleton(DirectoryScopes.ADMIN_DIRECTORY_GROUP)
+  val directoryScopes = Collections.singleton(DirectoryScopes.ADMIN_DIRECTORY_GROUP_MEMBER_READONLY)
```

Comment:
Good thought. But I don't think we should ever need a different scope unless we change the code to do something more than read. So we wouldn't really save ourselves any trouble by making this configurable.